backends-common/elasticsearch/src/main/java/org/apache/james/backends/es/ElasticSearchConstants.java

```diff
@@ -20,8 +20,9 @@
 package org.apache.james.backends.es;
 
 public interface ElasticSearchConstants {
-    String WRITE_ALIAS = "injectWriteAlias";
-    String READ_ALIAS = "injectReadAlias";
+    String MAILBOX_WRITE_ALIAS = "injectWriteAlias";
+    String MAILBOX_READ_ALIAS = "injectReadAlias";
 
     String MAILBOX_INDEX = "mailboxIndex";
+    String MAILBOX_MAPPING = "mailboxMapping";
```

Comment:
is there any reason to keep these mailbox related constants into common es backend?

Reply:
I would say `If everything went right there should not be some reasons not to do it.`